stubs/libusb_package/__init__.pyi

```diff
@@ -32,4 +33,4 @@ def find_library(candidate: str) -> Optional[str]: ...
 def get_libusb1_backend() -> Optional["IBackend"]: ...
 
 
-def find(*args: Any, **kwargs: Any) -> Any: ...
+def find(find_all: bool, custom_match: Optional[Callable[[usb.core.Device], bool]] = None) -> Iterable[usb.core.Device]: ...
```

Comment:
I don't think we should be changing this type signature. This was explicitly set in the library itself.

Reply:
This is a type stub that ensures we are passing valid arguments. It doesn't change the behaviour of the library, it just ensures that we can be sure of the types we are dealing with. 

Adding stubs over unspecified arguments isn't just standard, it's recommended best practice.